.eslintrc

```diff
@@ -1,25 +1,29 @@
 {
+    "extends": "eslint:recommended",
     "parser": "babel-eslint",
     "plugins": ["babel"],
     "rules": {
-        "strict": 0,
-        "indent": [2, 4],
-        "quotes": [2, "single"],
-        "linebreak-style": [2, "unix"],
+        "consistent-return": 2,
+        "eqeqeq": 2,
         "semi": [2, "always"],
-        "no-console": 0,
-        "babel/generator-star-spacing": 1,
-        "babel/new-cap": 1,
+
         "babel/array-bracket-spacing": 1,
+        "babel/arrow-parens": [1, "as-needed"],
```

Comment:
This is changed and controversial. This basically enforces parens when they are needed, and enforces a lack of parens when they are not needed. I could be convinced that this should be the other way.